spec/index.bs

```diff
@@ -1493,6 +1516,104 @@ interface GPUTextureUsage {
 };
 </script>
 
+### <dfn method for=GPUDevice>createTexture(descriptor)</dfn> ### {#GPUDevice-createTexture}
+
+<div algorithm="GPUDevice.createTexture">
+  **Arguments:**
+    - {{GPUTextureDescriptor}} |descriptor|
+
+  **Returns:** {{GPUTexture}}
+
+  1. If device is lost, or if this call doesn't follow the [$createTexture Valid Usage$], return an error texture.
+  1. Let |t| be a new {{GPUTexture}} object.
+  1. Set |t|.{{GPUTexture/[[textureSize]]}} to |descriptor|.{{GPUTextureDescriptor/size}}.
+  1. Set |t|.{{GPUTexture/[[sampleCount]]}} to |descriptor|.{{GPUTextureDescriptor/sampleCount}}.
+  1. Set |t|.{{GPUTexture/[[dimension]]}} to |descriptor|.{{GPUTextureDescriptor/dimension}}.
+  1. Set |t|.{{GPUTexture/[[format]]}} to |descriptor|.{{GPUTextureDescriptor/format}}.
+  1. Set |t|.{{GPUTexture/[[textureUsage]]}} to |descriptor|.{{GPUTextureDescriptor/usage}}.
+  1. Return |t|.
+
+  <div algorithm>
+    <dfn abstract-op>maximum mipLevel count</dfn>(dimension, size)
+    **Arguments:**
+        - {{GPUTextureDescriptor/dimension}} |dimension|
+        - {{GPUTextureDescriptor/size}} |size|
+
+    1. Calculate the values of |w|, |h|, and |d|. If the |dimension| is "1d":
+
+      1. Let |w| = |size|.[=Extent3D/width=].
+      1. Let |h| = 1.
+      1. Let |d| = 1.
+
+    1. Else if the |dimension| is "2d":
+
+      1. Let |w| = |size|.[=Extent3D/width=].
+      1. Let |h| = |size|.[=Extent3D/height=].
+      1. Let |d| = 1.
+
+    1. Else (the |dimension| is "3d"):
+
+      1. Let |w| = |size|.[=Extent3D/width=].
+      1. Let |h| = |size|.[=Extent3D/height=].
+      1. Let |d| = |size|.[=Extent3D/depth=].
+
+    1. Let |m| = the maximum value of |w|, |h|, and |d|.
+    1. Return one plus the greatest integral value of |x| for which 2^|x| <= |m|.
+  </div>
+
+  <div algorithm class=validusage>
+    <dfn abstract-op>createTexture Valid Usage</dfn>
+      Given a {{GPUDevice}} |this| and a {{GPUTextureDescriptor}} |descriptor|
```

Comment:
nit: indent level changes between these two lines.

Also I'd put a paragraph break between the dfn and "Given".

```suggestion
      <dfn abstract-op>createTexture Valid Usage</dfn>

      Given a {{GPUDevice}} |this| and a {{GPUTextureDescriptor}} |descriptor|
```